### ISSUE TITLE
ImportError using version 0.12.5

### ISSUE THREAD

**Issue by gabriel23cg** [jupyter-widgets/ipyleaflet#578]
Using Python 3.7.7 and IPython.
```python
from ipyleaflet import Map
ImportError: cannot import name 'Map' from 'ipyleaflet'
```

**Comment by davidbrochart** [comment]
Thanks for reporting the issue, it should be fixed in 0.12.6.

**Comment by SylvainCorlay** [comment]
We just released 0.12.6 !

Many thanks to @davidbrochart for fixing this on a holiday!
